docs/templates/layers/writing-your-own-keras-layers.md

```diff
@@ -34,4 +34,34 @@ class MyLayer(Layer):
         return (input_shape[0], self.output_dim)
 ```
 
+It is also possible to define Keras layers which have multiple input tensors and multiple ouput tensors. To do this, you should assume that the inputs and outputs of the methods `build(input_shape)`, `call(x)` and `compute_output_shape(input_shape)` are lists. Here is an example, similar to the one above:
+
+```python
+from keras import backend as K
+from keras.engine.topology import Layer
+import numpy as np
+
+class MyLayer(Layer):
+
+    def __init__(self, output_dim, **kwargs):
+        self.output_dim = output_dim
+        super(MyLayer, self).__init__(**kwargs)
+
+    def build(self, input_shape):
+        # Create a trainable weight variable for this layer.
+        self.kernel = self.add_weight(name='kernel',
```

Comment:
Could we make the assumptions more immediately readable, e.g.

assert isinstance(input_shape, list)
shape_a, shape_b = = input_shape
...
... shape=(shape_a[1], self.output_dim)